### PR TITLE
news: GRASS GIS docker images moved to OSGeo org

### PR DESCRIPTION
**Pull request by neteler**
GRASS GIS docker images moved from mundialis to OSGeo dockerhub organization.

Co-Authored-By: @mmacata

### CODE REVIEW

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+for the GRASS GIS docker images.
+
+<a href="https://hub.docker.com/r/osgeo/grass-gis/">
+  <img src="https://grass.osgeo.org/images/logos/grass-Docker.jpg"
+   alt="GRASS GIS docker images"
+   title="GRASS GIS docker images"
+   width="20%" style="float:right;padding-left:25px;padding-top:5px">
+</a>
+
+Historically, these docker images have been maintained and hosted under
```
Comment (veroandreo) [inline content/news/2023_08_27_GRASS_GIS_docker_organization.md:22]:
```suggestion
In the past years, these docker images have been maintained and hosted under
```

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+the mundialis organisation's repository. The company mundialis
+(https://www.mundialis.de) has played a crucial role in providing and
+maintaining these images, ensuring their availability and stability for
+the wider GIS community. The GRASS GIS development community extends
+its sincere appreciation to mundialis for their technical expertise and
+consistent support over the years.
+
+In a strategic move, the Docker images have now been moved to the OSGeo
```
Comment (veroandreo) [inline content/news/2023_08_27_GRASS_GIS_docker_organization.md:30]:
```suggestion
In a strategic move, the **GRASS GIS Docker images have now been moved** to the OSGeo
```

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+The Geographic Resources Analysis Support System
+([https://grass.osgeo.org/](/)), commonly referred to as GRASS GIS, is
+an Open Source Geographic Information System providing powerful raster,
+vector and [geospatial processing capabilities](https://grass.osgeo.org/learn/overview/).
+It can be used either as a stand-alone application, as backend for other
+software packages such as QGIS and R, or in the cloud. It is
+distributed freely under the terms of the GNU General Public License (GPL).
+GRASS GIS is a founding member of the Open Source Geospatial Foundation (OSGeo).
+
```
Comment (veroandreo) [inline content/news/2023_08_27_GRASS_GIS_docker_organization.md:58]: I'd remove this section, it's ok for releases, but I do not think we need it in news items.

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+This migration is not just a change of repository location. The move to
+the OSGeo repository has been accompanied by a systematic clean-up and
+reorganisation of the tags associated with the GRASS GIS docker images.
+This restructuring is intended to improve clarity, streamline
+versioning and optimise the user experience when selecting appropriate
+image tags for deployment.
+
+In summary, as we have phased out the repository under mundialis, we
+anticipate robust development, improved maintenance and a more
```
Comment (veroandreo) [inline content/news/2023_08_27_GRASS_GIS_docker_organization.md:44]:
```suggestion
anticipate improved maintenance and a more
```

robust development was and will still be there, and mundialis did a great job with maintenance as well... so, I dunno, this sentence seems weird to me

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+The Geographic Resources Analysis Support System
+([https://grass.osgeo.org/](/)), commonly referred to as GRASS GIS, is
+an Open Source Geographic Information System providing powerful raster,
+vector and [geospatial processing capabilities](https://grass.osgeo.org/learn/overview/).
+It can be used either as a stand-alone application, as backend for other
+software packages such as QGIS and R, or in the cloud. It is
+distributed freely under the terms of the GNU General Public License (GPL).
+GRASS GIS is a founding member of the Open Source Geospatial Foundation (OSGeo).
+
+_The GRASS Dev Team_
```
Comment (veroandreo) [inline content/news/2023_08_27_GRASS_GIS_docker_organization.md:59]: IMO the news can be signed by @neteler and @mmacata + GRASS dev team, we've been doing that before, and it is good practice to acknowledge those that actually contributed the work, time and text.

**Review by veroandreo — COMMENTED** [review]
I have left some comments here and there

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md (hunk 3 above)
Comment (neteler) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md]: Okay, dropped.

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md (hunk 5 above)
Comment (wenzeslaus) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md]: I agree. Let's turn this into a best practice. I would even say that generally "and the GRASS Development Team" is not necessary for a news items, but practically, other people (thanks @veroandreo and others!) help with the submission, so it makes sense to have this attribution when the contribution does not constitute full authorship.

**Review by wenzeslaus — COMMENTED** [review]
I suggest to consistently write Docker with an uppercase D. All lowercase is just the command on command line.

**Comment by neteler** [comment]
> I suggest to consistently write Docker with an uppercase D. All lowercase is just the command on command line.

Sure, done in 135a2fec60fd36c1760a636774f969e7bab2e6c8

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+under OSGeo GRASS GIS on Docker Hub:
+
+[https://hub.docker.com/r/osgeo/grass-gis/](https://hub.docker.com/r/osgeo/grass-gis/tags?page=1&ordering=-name)
+
+This migration is not just a change of repository location. The move to
+the OSGeo repository has been accompanied by a systematic clean-up and
+reorganisation of the tags associated with the GRASS GIS Docker images.
+This restructuring is intended to improve clarity, streamline
+versioning and optimise the user experience when selecting appropriate
+image tags for deployment.
```
Comment (echoix) [inline content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: I would have expected either a summary of changes or the actions needed to react to this change, or a link to where it is explained.

For example, it could be that now we recommend using osgeo/grass-gis:xxxxx tag for most users, and osgeo/grass-gis:yyyy tag for others to pin a specific version.

Or it could be a suggestion based on the tag used before and the new tag series that should be mapped to.

Comment (neteler) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: You mean to add a kind of lookup table?

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md (hunk 4 above)
Comment (neteler) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md]: Agreed.
I have rephrased it but the closing should be improved. Suggestions welcome.

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md (hunk 6 above)
Comment (echoix) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: It can be more simple than that, but at least to answer the question: As a user of the old images, what should I do to use the new organisation and tags?

Comment (neteler) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: Perhaps @mmacata can assist here?

Comment (neteler) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: For the related discussion, see https://github.com/OSGeo/grass/pull/3075

Comment (echoix) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: While looking for this, I observed that the 7.8.8 release on GitHub is marked as the "latest" release for GitHub, while it should have stayed 8.3.0, as 7.8.8 was a backport. It should be possible to go to the 8.3.0 release, edit it, and check the "Set as the latest release" checkbox even if it was older.
image

Comment (neteler) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: > While looking for this, I observed that the 7.8.8 release on GitHub is marked as the "latest" release for GitHub, while it should have stayed 8.3.0, as 7.8.8 was a backport.

Ouch. Fixed (8.3.0 is again latest).

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+In summary, as we moved the repository from mundialis to the OSGeo
+organisation, we took the opportunity to clean up the Docker tags.
+
+_Carmen Tawalika (mundialis), Markus Neteler and The GRASS Dev Team_
```
Comment (mmacata) [inline content/news/2023_08_27_GRASS_GIS_docker_organization.md:46]:
```suggestion

Now there are four different types of tags, while `{{ os }}` can be replaced with "alpine", "debian" or "ubuntu" (and for more recent versions also with "ubuntu_wxgui"):
- `latest` for current releasebranch (at this time 8.3) and with ubuntu operating system
- `current-{{ os }}` for current releasebranch (at this time 8.3), e.g `current-ubuntu`
- `{{ branch }}-{{ os }}` for all configured branches, e.g. `releasebranch_8_3-alpine`,  `main-debian`
- `{{ tag }}-{{ os }}` for all published releases. e.g. `8.3.0-alpine`

While naming of branches and tags is still the same, it adds `current-{{ os }}` and abandons deprecated naming style `latest-{{ os }}` and `stable-{{ os }}`. Also content of `latest` tag was changed from `main` branch to current releasebranch.

If `mundialis/grass-py3-pdal:latest-{{ os }}` was used containing the latest development version, it should now be replaced with `osgeo/grass-gis:main-{{ os }}`. For `mundialis/grass-py3-pdal:stable-{{ os }}` (which was formerly build upon releasebranch_7_8) a switch to the current stable releasebranch `osgeo/grass-gis:current-{{ os }}` is recommended or the usage of a specific releasebranch `releasebranch_8_3-{{ os }}`. Old GRASS GIS Dockerimages with version 7.8 can still be found at mundialis/grass-py3-pdal.

In summary, as we moved the repository from mundialis to the OSGeo
organisation, we took the opportunity to clean up the Docker tags.

Related docker build configuration changes can be found here:
- [docker: Update dockerhub organization](https://github.com/OSGeo/grass/pull/3001)
- [docker: CI update and simplification](https://github.com/OSGeo/grass/pull/3075)

_Carmen Tawalika (mundialis), Markus Neteler and The GRASS Dev Team_
```

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md (hunk 6 above)
Comment (mmacata) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: I added a more detailed description as suggestion below which should make it more easy to decide how to migrate.
I am not sure if the language fits because some text was a copy & paste from the PR description :upside_down_face:

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md (hunk 7 above)
Comment (echoix) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:46]: LGTM!

File: content/news/2023_08_27_GRASS_GIS_docker_organization.md (hunk 6 above)
Comment (neteler) [reply content/news/2023_08_27_GRASS_GIS_docker_organization.md:41]: Thanks, this new part now looks like this:

image

**Review by echoix — APPROVED** [review]
